meshtastic/tests/test_ble_interface.py

```
@@ -0,0 +1,339 @@
+    iface = BLEInterface()
+    iface._closed = True
+    iface._event_loop = MagicMock()
+    
+    # Should not create a task when interface is closed
+    iface.from_num_handler(None, b'\x01\x00\x00\x00')
+    iface._event_loop.create_task.assert_not_called()
+    iface.close()
+
+
+@pytest.mark.ble
+def test_from_num_handler_with_lock(mocker):
```

Ruff (0.13.1)

Unused function argument: `mocker`

(ARG001)